src/reports.py

```diff
@@ -0,0 +1,20 @@
+import jinja2
+import os
+
+
+class ReportGenerator(object):
+
+    def __init__(self):
+        this_dir = os.path.dirname(os.path.abspath(__file__))
+        template_dir = os.path.join(this_dir, 'templates')
+        loader = jinja2.FileSystemLoader(template_dir)
+        self.jinja_env = jinja2.Environment(loader=loader)
+
+    def render_report(self):
+        template = self.jinja_env.get_template('base.html')
+        print(template.render(pull_requests=[]))
```

Comment:
Should we instead dump the results to an `.html` file so that it can be directly opened and viewed in a browser?